src/components/layouts/featurelist/feature-list-content.tsx

```diff
@@ -0,0 +1,538 @@
+              identifiers.
+            </>
+          ),
+          icon: CheckCircle
+        },
+        {
+          title: 'Global Access Control',
+          description: (
+            <>
+              Coming soon: Deploy around the globe and provide a fast response
+              no matter where your users are.
+            </>
+          ),
+          icon: PlusCircle
+        },
+        {
+          title: 'Strong Consistency',
+          description: (
+            <>
+              Coming soon: Provide a straightforward user experience by serving
+              strong consistent responses using{' '}
+              <Button
+                style={'link-inline'}
+                to={'https://github.com/ory/keto/issues/517'}
+              >
+                snaptokens
+              </Button>
+              .
+            </>
+          ),
+          icon: PlusCircle
+        }
+      ]
+    }
+  ]
+}
+
+export const featureListOathkeeper = {
+  features: [
+    {
+      icon: CircleThreePlus,
+      title: 'Ory Oathkeeper Features',
+      description: (
+        <>
+          Forward authorized requests and reject unauthorized ones. Ory
+          Oathkeeper can act as a reverse proxy or interface with your existing
+          API Gateway.
+        </>
+      ),
+      button: (
+        <Button
+          style={'link'}
+          to={'/oathkeeper/docs/configure-deploy/'}
+          iconRight={ArrowRight}
+        >
+          Get started
+        </Button>
+      ),
+      features: [
+        {
+          title: 'Identity and Access Proxy',
+          description: (
+            <>
+              Authorize HTTP requests to your server based on sets of Access
+              Rules.
+            </>
+          ),
+          icon: CheckCircle
+        },
+        {
+          title: 'Zero Trust Architecture',
+          description: (
+            <>
+              Apply Zero Trust / BeyondCorp security in your tech stack. Ory
+              Oathkeeper enables secure access for all users (e.g. employees,
+              partners, customers).
+            </>
+          ),
+          icon: CheckCircle
+        },
+        {
+          title: 'Carefree Integration',
+          description: (
+            <>
+              Deploy alongside your existing API Gateways. Ory Oathkeeper plugs
+              into Ambassador, Envoy, AWS API Gateway, Nginx and many more.
+            </>
+          ),
+          icon: CheckCircle
+        },
+        {
+          title: 'Enrich Data',
+          description: (
+            <>
+              Accumulate data from multiple sources and parse it into custom
+              headers: X-User-ID, JSON Web Tokens and more.
+            </>
+          ),
+          icon: CheckCircle
+        },
+        {
+          title: 'Next Generation Oathkeeper',
+          description: (
+            <>
+              <i>Coming soon:</i> The next generation of Ory Oathkeeper. Learn
```

Comment:
No `<i>` tags, please use `<em>` instead:

```suggestion
              <em>Coming soon:</em> The next generation of Ory Oathkeeper. Learn
```